components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -909,8 +909,8 @@
     } else if (isProvisioningEnabled && provisioningUserStoreId == null) {
         provisionStaticDropdownDisabled = "disabled=\'disabled\'";
     }
-
-    userStoreDomains = client.getUserStoreDomains();
+    userStoreDomains.addAll(Arrays.asList(client.getUserStoreDomains()));
+    userStoreDomains.add("As in username");
```

Comment:
why don't we use the constant that has already defined.

Reply:
Fixed with 3fd10782c5c720e39d5ff317a696484b155378de